apps/api/e2e/tests/admin.test.ts

```diff
@@ -36,19 +34,8 @@ describe("Admin API", () => {
 
   // Clean up test state before each test
   beforeEach(async () => {
-    await cleanupTestState();
```

Comment:
Calling this in individual test files doesn't make sense.  We can just do it once in the test-setup.ts file and make sure it happens for every test.